portlets/src/main/webapp/vue-app/engagement-center/components/programs/ProgramDetail.vue

```diff
@@ -26,18 +26,18 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
       </v-btn>
       <span class="text-header-title ps-1"> {{ programTitle }} </span>
       <v-spacer />
-      <span class="text-header-title d-none d-md-block" v-sanitized-html="$t('programs.budget', $t(programBudgetLabel))"></span>
+      <span class="text-header-title" v-sanitized-html="$t('programs.budget', $t(programBudgetLabel))"></span>
```

Comment:
Please ensure that the work done [here](https://github.com/Meeds-io/gamification/commit/fc38806b842d62c5be3fcc6a93c54f37aea1cc24) is not reverted